openedx/features/assessment/helpers.py

```diff
@@ -29,13 +29,18 @@ def can_auto_score_ora(enrollment, course, block, index_chapter):
     anonymous_user = get_anonymous_user(enrollment.user, course.id)
     if not anonymous_user:
         return False
-    response_submission = Submission.objects.filter(
+    response_submissions = Submission.objects.filter(
         student_item__student_id=anonymous_user.anonymous_user_id,
         student_item__item_id=block
-    ).order_by('submitted_at').first()
+    ).order_by('submitted_at')
+    log.info('Multiple submissions found having ids {ids} in can_auto_score_ora'.format(
```

Comment:
we haven't put any check in place to see if there are actually multiple records exists here. I think this log should be inside if multiple records exists check passes. Please check for this in other places in the code as well.

Reply:
Good point 👍